.github/workflows/build-amdvlk-docker.yml

```diff
@@ -34,13 +34,12 @@ jobs:
         run: |
           git clone https://github.com/${GITHUB_REPOSITORY}.git .
           git checkout ${GITHUB_SHA}
-      - name: Setup Google Cloud SDK
-        uses: google-github-actions/setup-gcloud@master
+      - id: 'auth'
```

Comment:
Can we also give it a name like "Authenticate into Google Cloud" or similar?

Reply:
Sure, done.